pkg/config/config.go

```diff
@@ -117,6 +124,10 @@ type NsxConfig struct {
 	EnvoyHost                 string   `ini:"envoy_host"`
 	EnvoyPort                 int      `ini:"envoy_port"`
 	LicenseValidationInterval int      `ini:"license_validation_interval"`
+	UseAVILB                  bool     `ini:"use_avi_lb"`
+	UseNativeLoadBalancer     *bool    `ini:"use_native_loadbalancer"`
```

Comment:
Why it is *bool?

Reply:
Wont be better to add a LB type field which can be set to either "AVI" or "NSX"?

Reply:
UseNativeLoadBalancer default value is true, thus we need use *bool to check "not set".

Added `NSXLBEnabled` function for this.

Reply:
When we have a case both AVI LB and NSX LB should be set to false? If not such cases, why we need two parameters?

If we need to keep this one, let us rename it to `UseNSXLoadBalancer`

Reply:
NCP already has these 2 options and uses a function to check the LB type.

Changed option name.